apps/ledger-live-desktop/src/renderer/screens/exchange/Swap2/Form/SwapWebView.tsx

```diff
@@ -77,6 +93,53 @@ const SwapWebView = ({ swapState, redirectToProviderApp }: SwapWebProps) => {
         onSwapWebviewError(params);
         return Promise.resolve();
       },
+      "custom.saveSwapToHistory": (swap: SwapProps, transaction_id: string) => {
+        if (
+          !swap ||
+          !transaction_id ||
+          !swap.fromTokenId ||
+          !swap.provider ||
+          !swap.toTokenId ||
+          !swap.fromAmountWei ||
+          !swap.toAmountWei
```

Comment:
Perhaps wei may not make sense here? If it's and evm coin/token?

Reply:
Yeah it did not make sense. I removed it and used currency magnitude inside the live-app to compute it